src/parseNode.js

```diff
@@ -418,30 +370,19 @@ type ParseNodeTypes = {
         type: "phantom",
         mode: Mode,
         loc?: ?SourceLocation,
-        value: {|
-            type: "phantom",
-            value: AnyParseNode[],
-        |},
+        body: AnyParseNode[],
     |},
     "hphantom": {|
         type: "hphantom",
         mode: Mode,
         loc?: ?SourceLocation,
-        value: {|
-            type: "hphantom",
-            body: AnyParseNode,
-            value: AnyParseNode[],
-        |},
+        body: AnyParseNode,
     |},
     "vphantom": {|
         type: "vphantom",
         mode: Mode,
         loc?: ?SourceLocation,
-        value: {|
-            type: "vphantom",
-            body: AnyParseNode,
-            value: AnyParseNode[],
-        |},
+        body: AnyParseNode,
```

Comment:
Why don't we need the `AnyParseNode[]` that was originally on the phantoms?

Reply:
`value = ordargument(body)`. There was no reason to store both.

Plus I've gotten allergic to anything named `value`. :) We should probably rename the rest to have names that say what they are.